YouVsVirus/Assets/Scripts/Components/Player.cs

```diff
@@ -44,15 +49,21 @@ public override void SetCondition(int condition)
 		{
 			base.SetCondition(condition);
 
-			if(condition == EXPOSED)
+			if (condition == EXPOSED)
 			{
 				LevelSettings.GetActiveEndLevelController().NotifyPlayerExposed();
 			}
-			if(condition == DEAD)
+			if (condition == DEAD)
 			{
 				LevelSettings.GetActiveEndLevelController().NotifyPlayerDied();
 			}
 		}
+		public void Update()
+		{
+			if(infectedByPropaganda)
+				LevelSettings.GetActiveEndLevelController().NotifyPlayerInfectedByPropaganda();
```

Comment:
Is there a reason why this is not solved via SetCondition?

Reply:
Yes, SetCondition is called by probabiblity based infecttion
if (infectionControl.IsNewDay())
            {
                UpdateCondition();
            }
only on a NewDay(). The Canvas, however, should appear as soon as the player is infected.